### PR TITLE
Migrate monitoring jobs

### PR DESCRIPTION
**Pull request by josephperrott**

### CODE REVIEW

**Comment by pkozlowski-opensource** [comment]
@josephperrott looks like this PR still needs a bit of cleanup (lint job is failing)

File: .github/workflows/monitoring.yml [hunk 1]
```diff
@@ -0,0 +1,54 @@
+name: Monitoring
+
+on:
+  schedule:
+    - cron: '0 10 * * *'
```
Comment (devversion) [inline .github/workflows/monitoring.yml:5]: Do you want to add a comment like in Circle?

File: .github/workflows/monitoring.yml [hunk 2]
```diff
@@ -0,0 +1,54 @@
...
+
+permissions: {}
+
+defaults:
+  run:
+    shell: bash
+
+jobs:
+  check_aio_production:
```
Comment (devversion) [inline .github/workflows/monitoring.yml:14]: Nit: Do we just want to name this `check_aio_deployments`? We are not deploying any canary/staging so production seems to be more confusing than helpful IMO

**Review by devversion — APPROVED** [review]
Nice, that's so clean!

**Comment by AndrewKushnir** [comment]
(FYI, changing the label from "merge" -> "review", since there is some feedback from Paul)

File: aio/content/marketing/contributors.json [hunk 3]
```diff
@@ -785,14 +782,12 @@
     "name": "Nishu Goel",
     "picture": "nishugoel.jpg",
     "twitter": "TheNishuGoel",
-    "website": "https://unravelweb.dev",
```
Comment (MarkTechson) [inline aio/content/marketing/contributors.json:788]: This URL is not dead.

**Review by MarkTechson — CHANGES_REQUESTED** [review]
we should keep the one that wasn't dead, I commented.

File: aio/content/marketing/contributors.json (hunk 3 above)
Comment (josephperrott) [reply aio/content/marketing/contributors.json]: reverted

**Comment by AndrewKushnir** [comment]
(FYI, removing the "merge" label since the PR needs a re-review from Mark)

**Review by MarkTechson — APPROVED** [review]
LGTM

**Comment by jessicajaniuk** [comment]
This PR was merged into the repository by commit e6235accb11fd63854d37c910212b5125ef435e4.

**Pull request closed without merging**